frontend/content/faq.json

```diff
@@ -7,10 +7,6 @@
     "label": "Co to jest \"flaga\"?",
```

Comment:
You can also add sections like `Kiedy nowa edycja?`, `Jak mogę wziąć udział?`, `Jakie były edycje do tej pory?`